examples/preprocessing/plot_rereference_eeg.py

```diff
@@ -35,7 +35,7 @@
 
 reject = dict(eog=150e-6)
 epochs_params = dict(events=events, event_id=event_id, tmin=tmin, tmax=tmax,
-                     picks=picks, reject=reject)
+                     picks=picks, reject=reject, proj=True)
```

Comment:
it is easy to miss this if it is only here, and to a newcomer it may not be intuitive that `proj=True` relates to applying reference.